recipe/meta.yaml

```
@@ -35,6 +35,7 @@ requirements:
   run:
     # CNI version from singularity/Gopkg.lock
     - cni-plugins >=0.7.4  # [linux]
+    - gocryptfs  # [linux]
```

You need to ensure this only is dependeng on for the architectures `gocryptfs` is published for. I'm not 100% sure my syntax is correct. [You can refer to this this](https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#preprocessing-selectors).
```suggestion
    - gocryptfs  # [(linux or osx) and x86_64]
```